ref/glossary.md

```diff
@@ -135,6 +141,21 @@ A self-contained deployment variant of Anbox Cloud.
 
 See [Variants](https://discourse.ubuntu.com/t/about-anbox-cloud/17802#variants).
 
+<a name="anbox-cloud-cluster"></a>
+### Anbox Cloud cluster
+
+A deployment of the Anbox Cloud, either just the core stack or the core stack along with the streaming stack.
+
+<a name="anbox-cloud-subcluster"></a>
+### Anbox Cloud subcluster
+
+The group of components that is made up of the management layer (AMS, AMC, etcd), control node, LXD, AMS node controller.
```

Comment:
I am not quite sure if we wanna mention the control node here next to `the management layer` as they seem to be the same to me.